projects/storefrontapp/src/environments/models/build.process.env.d.ts

```diff
@@ -11,4 +11,5 @@ interface Env {
   CX_B2B: boolean;
   CX_CPQ: boolean;
   CX_DIGITAL_PAYMENTS: boolean;
+  CX_OLD_CHECKOUT: boolean;
```

Comment:
should add this under `.env-cmdrc` for "b2c" and "b2b" in case we want to use the old checkout instead

Reply:
Thanks, added as
`"CX_OLD_CHECKOUT": "false"`